firebase.json

```diff
@@ -11,7 +11,7 @@
       "port": 5001
     },
     "firestore": {
-      "port": 8080
+      "port": 6000
```

Comment:
How was this port number decided?

I see algolia's plugin has the following configuration:
```json
  "firestore": {
      "port": 8080
    },
    "ui": {
      "enabled": true,
      "port": 4000
    }
```

Reply:
It's the configuration for the emulator. I can change it if you want

Reply:
No I was just wondering why you chose `6000` specifically and not `8080` for example

Reply:
No reason but you're right I think the auto-generated `firebase.json` has `8080` I will change it